src/App.vue

```diff
@@ -88,6 +88,7 @@ body {
   margin: 0;
   font-size: 87.5%;
   letter-spacing: 0.04em;
+  overflow-y: scroll;
```

Comment:
this must be set for `html` to work afaik.

Reply:
it worked for me on body, but `html` seems to be the safer option